app/actions/article.js

```diff
@@ -29,9 +29,12 @@ export const REMOVE_ARTICLE_SUCCESS = 'REMOVE_ARTICLE_SUCCESS';
 export const REMOVE_ARTICLE_FAILURE = 'REMOVE_ARTICLE_FAILURE';
 
 
+const toJS = (x) => x.toJS ? x.toJS() : x;
```

Comment:
I would avoid the method name shadowing and rename it to: `convertToNativeJavascript`